src/main.js

```diff
@@ -58,7 +58,7 @@ if (!window.SplunkRum) {
     if (!options.debug) {
       if (!options.beaconUrl) {
         throw new Error('SplunkRum.init( {beaconUrl: \'https://something\'} ) is required.');
-      } else if(!options.beaconUrl.startsWith('https') && !options.allowInsecureBeacon) {
+      } else if(options.beaconUrl.startsWith('http:') && !options.allowInsecureBeacon) {
```

Comment:
Without `allowInsecureBeacon` we shouldn't allow a relative `beaconUrl: '/beacon'` unless it resolves to an https url.

Reply:
as discussed in Slack, the `allowInsecureBeacon` part is removed from this PR